web/locales/cs.yml

```diff
@@ -0,0 +1,6 @@
+cs:
+  RecurringJobs: Pravidelně opakované
+  Name: Jméno
```

Comment:
Sorry, I thought `Class` and `Queue`... It's in original yaml already translated... I don't mind to rewrite it, but I think that it would be fine to hold `Class` and `Queue` translated as is in sidekiq repository.